### PR TITLE
[W08H01] adding the test if one collection out of the array is empty

### PR DESCRIPTION
**Pull request by Benn1x**
i added for the exercise intersection the test if one Collection out of the array is empty the return collection is also empty

### CODE REVIEW

**Comment by ipadmini22** [comment]
I am not sure that Homework exists ;)

File: w08h01/test/pgdp/SimpleGenericsTest.java [hunk 1]
```diff
@@ -141,6 +141,22 @@ void intersection2() {
         Collection<ArrayList>[] listArray = new Collection[]{list1, list, list3};
         assertEquals("[1, 2, 4]", Arrays.toString(SimpleGenerics.intersection(listArray).toArray()));
     }
+    @Test
```
Comment (nilsreichardt) [inline w08h01/test/pgdp/SimpleGenericsTest.java:144]:
```suggestion

    @Test
```

Leave an empty between the tests

File: w08h01/test/pgdp/SimpleGenericsTest.java [hunk 2]
```diff
@@ -141,6 +141,22 @@ void intersection2() {
...
+    @DisplayName("intersection collections array, with one empty collection")
+    void intersection2() {
```
Comment (nilsreichardt) [inline w08h01/test/pgdp/SimpleGenericsTest.java:146]: There is already a method called `intersection2`.

```suggestion
    void intersectionWithEmptyList() {
```

Maybe this a better method name?

**Review by nilsreichardt — CHANGES_REQUESTED** [review]
Thanks for writing this test! :)

From my side only the two comments, otherwise LGTM. And you should rename the title to W08H01 :D

**Comment by Benn1x** [comment]
I'm so sorry that i i said [W08H08] i meant [W08H01] sorry

**Comment by Benn1x** [comment]
i guess now should be everything fine

**Review by nilsreichardt — APPROVED** [review]
Yeah, now it looks good! 😊

**Review by MaximilianAnzinger — COMMENTED** [review]
🐧
